packages/boot/test/bootstrapTests/lca.test.ts

```diff
@@ -89,4 +89,26 @@ test.serial('stakeBld', async t => {
       },
     },
   });
+  t.like(wd.getLatestUpdateRecord(), {
+    status: { id: 'request-delegate', numWantsSatisfied: 1 },
+  });
+
+  await t.throwsAsync(
+    wd.executeOffer({
```

Comment:
what is the expected error and why?  Seems like somethign we shoudl check the details of, given that there are so many layers that could fail

Reply:
The expected error was commented out at the end of the function (this is the typical place to assert error message values in an ava test).

Inspection of the error message is disabled due to this: https://github.com/Agoric/agoric-sdk/pull/9591/files#r1655705785 - still not quite sure the past way to propagate an error through this bridge device.

The test still serves value since the rejection is properly propagated - just the message is missing.

I have confidence about the layering via tests in `bootstrapTests/orchestration.test.ts`, which show some "ABCI code: 5: error handling packet: see events for details", as well as unit tests for local-orch-acct.